src/proxy/query_test.go

```diff
@@ -733,3 +733,44 @@ func TestProxyQueryNotSupport(t *testing.T) {
 	expected := "unsupported.query:repair (errno 1105) (sqlstate HY000)"
 	assert.EqualValues(t, expected, actual.Error())
 }
+
+func TestProxyLowerCase(t *testing.T) {
```

Comment:
Here we need more lowercase test cases, such as:
insert/update/delete and other DML

Reply:
> Here we need more lowercase test cases, such as:
> insert/update/delete and other DML

done